public/js/helpers/humanize_helper.js

```diff
@@ -124,6 +124,12 @@ var humanize = {
       return result
     }
     return Math.floor(seconds) + 's'
+  },
+  formatTxDate: function (stamp, withTimezone) {
+    var d = new Date(stamp)
```

Comment:
This might need to be changed to:
`var d = new Date(stamp) * 1000`

Reply:
It seems like this outdated argument name is a holdover from [here](https://github.com/decred/dcrdata/commit/de314ca52fec01026ae08dfef44a56ed838f4909). It should probably be changed to something more accurate for the data type being passed. Maybe something like `timeDef` to mirror the type name server-side?